src/components/Input.js

```diff
@@ -1,9 +1,5 @@
 import { useState } from "react";
-import {
-  InputContainer,
-  Label,
-  Textarea,
-} from "/src/components/styles/InputStyle.js";
+import { Label, Textarea } from "/src/components/styles/InputField.js";
```

Comment:
Same here, use the prefix "Styled" and also your whole styles folder structure is pretty messy, you import Label and Textarea from a file called InputField.js. Create a separate file for each styled component that you reuse. Right now your your use of styledComponents is inconsistent and hard to understand. Checkout the handouts for styled components I and II.